dmscripts/email.py

```diff
@@ -0,0 +1,17 @@
+import io
+from jinja2 import Template
+
+from mandrill import Mandrill
+
+
+def render_html(template_path, data=None):
```

Comment:
Does `render_html` belong in `email`?  It seems more general purpose, and e.g. looks odd to import something from `email` in the `generate_agreement_signature_pages` script, which doesn't do anything with emails at all.